mlir/lib/Analysis/Liveness.cpp

```diff
@@ -72,6 +72,11 @@ struct BlockInfoBuilder {
         defValues.insert(result);
       for (Value operand : op->getOperands())
         useValues.insert(operand);
+      for (Region &region : op->getRegions())
+        for (Block &child : region.getBlocks())
+          for (BlockArgument arg : child.getArguments()) {
+            defValues.insert(arg);
+          }
```

Comment:
Nit: If you need braces, it should be for the outer loop I believe, not the inner ones.

Reply:
Personally I eschew braces anywhere possible since my parents couldn't afford orthodontics when I was a kid...